deps/npm/package.json

```diff
@@ -172,7 +172,7 @@
     "deep-equal": "~1.0.0",
     "marked": "~0.3.3",
     "marked-man": "~0.1.5",
-    "nock": "~2.7.0",
+    "nock": "~2.9.1",
```

Comment:
We don't include `devDependency` stuff in the distribution, but still bump the versions in `package.json`. Is that what you meant?